HW-0.Rmd

```diff
@@ -21,9 +21,12 @@ completion. Your analysis is now online! You can update what's online by
 repeating steps 2 and 3.
 1. Copy the URL of the resulting RPubs page.
 1. Replace the https://www.nhl.com/ URL below with the URL of your RPubs page:  
-[My RPubs Document](https://www.nhl.com/)
+[My RPubs Document](http://rpubs.com/khobbs/210264)
 1. Indicate how much time you spent on this assignment up to now, 
 including forking the `HW-0` repo and setting up the RStudio project: 
+
+1 hour. Spent lots of time figuring out how to open Git through RStudio.
```

Comment:
This is a little longer than I would've liked; this is not a class on GitHub after all. Next time you feel you're getting hung up on GitHub stuff, please speak to me. I'm flexible on submission deadlines in situations like these.